internal/handlers/handler_firstfactor.go

```diff
@@ -95,7 +95,8 @@ func FirstFactorPost(msInitialDelay time.Duration, delayEnabled bool) middleware
 
 		if err != nil {
 			ctx.Logger.Debugf("Mark authentication attempt made by user %s", bodyJSON.Username)
-			ctx.Providers.Regulator.Mark(bodyJSON.Username, false) //nolint:errcheck // TODO: Legacy code, consider refactoring time permitting.
+
+			_ = ctx.Providers.Regulator.Mark(bodyJSON.Username, false)
```

Comment:
Why not handle the error here?

Reply:
I had issues when running integration tests and I thought it was related.

After some testing it appears that it's not so I'll handle these too.

Reply:
Addressed in https://github.com/authelia/authelia/pull/1511/commits/15646c4bedd408da2fe42ddef2c07318c3a54c25.